math_keyboard/lib/src/foundation/math2tex.dart

```diff
@@ -69,12 +68,26 @@ List<TeX> _convertToTeX(Expression mathExpression, TeXNode parent) {
         ),
       ];
     }
-    // Note that modulo is unsupported.
-    throw UnimplementedError();
+    if (result == null) {
+      // Note that modulo is unsupported.
+      throw UnimplementedError();
+    }
+    // Wrap with parentheses to keep precedence.
+    return [
+      TeXLeaf('('),
+      ...result,
+      TeXLeaf(')'),
+    ];
   }
   if (mathExpression is Literal) {
     if (mathExpression is Number) {
       final number = mathExpression.value as double;
+      if (number == math.pi) {
+        return [TeXLeaf(r'\pi')];
+      }
+      if (number == math.e) {
+        return [TeXLeaf('e')];
+      }
```

Comment:
This is gold! 😍 

Can we add test cases for this one in particular? I think they can be a lot better than the existing setup as well (which I just copied from the other "end to end", as in string to expression to string to conversion).

I am asking because I am wondering if we every have rounding errors / inaccuracies with this one 🤔